documentation/WiFi_Direct_Support.md

```diff
@@ -0,0 +1,18 @@
+## Wi-Fi Direct Support
```

Comment:
Should this be linked from somewhere else?  (see if we do that for the mdns docs)